src/providers/Comscore.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Comscore
+ * https://direct.comscore.com/clients/help/FAQ.aspx#faqTagging
+ *
+ * @class
+ * @extends BaseProvider
+ */
+
+class ComscoreProvider extends BaseProvider {
+  constructor() {
+    super();
+    this._key = "COMSCORE";
+    this._pattern = /sb\.scorecardresearch\.com.+(?<!\.js)$/;
+    this._name = "Comscore";
+    this._type = "marketing";
+  }
+
+  /**
+   * Retrieve the column mappings for default columns (account, event type)
+   *
+   * @return {{}}
+   */
+  get columnMapping() {
+    return {
+      account: "c2",
+      requestType: "requestType"
```

Comment:
I would change this to `c1` since it already exists in the `keys()` :)

Reply:
I had the request type mapped in the column as c1, but seeing the "2" looks weird IMO, but I guess if thats would it is, it doesn't matter. Comscore's docs literally say that the c1 parameter is required and describes the event type, but is "prepopulated with a constant value of 2"
<img width="619" alt="Screen Shot 2019-04-30 at 8 20 19 PM" src="https://user-images.githubusercontent.com/8012764/57001304-ef3bb200-6b85-11e9-9840-8a1961a819c0.png">